specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -3039,6 +3069,10 @@
             "enabled": {
               "type": "boolean",
               "description": "Indicates whether the metadata sync is enabled or disabled"
+            },
+            "syncIntervalInMinutes": {
+              "type": "integer",
```

Comment:
Can you please specify format here. BTW I couldn't find this field in SQL's swagger spec.

Reply:
Added.